app/models/role.rb

```diff
@@ -0,0 +1,10 @@
+class Role < ActiveRecord::Base
+  has_and_belongs_to_many :users, :join_table => :users_roles
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer `has_many :through` to `has_and_belongs_to_many`.